pkg/kv/kvclient/kvcoord/dist_sender_rangefeed_canceler_test.go

```diff
@@ -12,46 +12,45 @@
 package kvcoord
 
 import (
+	"context"
 	"sync/atomic"
 	"testing"
 	"time"
 
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
+	"github.com/cockroachdb/errors"
 	"github.com/stretchr/testify/require"
 )
 
-type cancelRec int32
-
-func (c *cancelRec) cancel() {
-	atomic.StoreInt32((*int32)(c), 1)
-}
-
-func (c *cancelRec) canceled() bool {
-	return atomic.LoadInt32((*int32)(c)) != 0
-}
-
 func TestStuckRangeFeedCanceler(t *testing.T) {
```

Comment:
We should add another test that makes sure it won't fire when the processing time between `do()` calls is much greater than the threshold (e.g. 2x).